### DISCUSSION TITLE
How can I deal the editor canncel the text-indent

### DISCUSSION THREAD

**Discussion by Tim-story** [ueberdosis/tiptap#2309] [General Questions]
when i give it value frist time like '<p>  hello everyone</p>' it is normal but when I give it value second time like '<p>  hello <span style="color:red;>everyone</span>" that my text-indent is clear on the page (vue2.X  v-model)

**Comment by philippkuehn** [comment]
Not sure what you mean. Please provide any code example so someone can help you.
